src/components/Login.jsx

```diff
@@ -24,23 +24,34 @@ function login(user) {
         .then((data) => data.json());
   }
 }
+const validationSchema = Yup.object({
+  email: Yup
+      .string()
+      .email('Invalid email format')
+      .required('Email is required'),
+  password: Yup
+      .string()
+      .required('Password is required'),
```

Comment:
this is great! I'd also recommend adding  a `.default('')` to each, which you can use to create initial values with `schema.default()`